hack/use-openshift-prometheus.sh

```diff
@@ -216,6 +216,12 @@ EOM
   fi
 
   echo "Apply Telemetry resource to [${ISTIO_NAMESPACE}] to ensure Prometheus is a metrics provider"
+  tel_name=$(${OC} get telemetry -n ${ISTIO_NAMESPACE} -o name)
+  if [ ! -z "$tel_name" ]
+  then
+    echo "Found existing ${tel_name} in ${ISTIO_NAMESPACE}, patching..."
```

Comment:
Wrap in brackets the dynamic text to be displayed so it is easy to see in the output what is static text and what is dynamic.
```suggestion
    echo "Found existing [${tel_name}] in [${ISTIO_NAMESPACE}], patching..."
```